CHANGELOG.md

```diff
@@ -12,7 +12,8 @@
   changed format, with the addition of the `messageExpiry` field.
 
 ## Mobile wallet
-
+- the randomness output by `create_credential` is now also output by `create_credential_aux`
+  so that the randomness can be stored. 
```

Comment:
Can you please name the exact field that is added to the response of this call in the wallet library.

It would also be good to update the example files in `rust-bins/wallet-notes` with the new response format.